db/.env.example

```diff
@@ -4,4 +4,7 @@ DB_PASSWORD=
 DB_HOST=
 DB_PORT=
 SESSION_SECRET=
-SLACK_BOT_TOKEN=
\ No newline at end of file
+SLACK_BOT_TOKEN=
+
+SLACK_TEST=false
```

Comment:
In this file it's `SLACK_TEST` and `SLACK_TEST_TOKEN`, but the testing configuration is looking for `TEST_SLACK` and `TEST_SLACK_TOKEN`. Can you swap those around?

Reply:
Should be fixed!